streams/src/main/java/org/apache/kafka/streams/TopologyConfig.java

```diff
@@ -217,8 +217,19 @@ public TopologyConfig(final String topologyName, final StreamsConfig globalAppCo
         }
     }
 
+    public static Materialized.StoreType parse(final String storeType) {
```

Comment:
Is this mentioned on the KIP? Looks like a public API change to me.

Reply:
🤦 I forgot this is public too, it doesn't need to be. Any suggestions of where to put it?

Reply:
Moved to `MaterializedInternal` for it's final resting place 💀

Reply:
I hate that we moved this into a public package and made it a public API (even though I'm sure I ok'ed it at the time) 😞 

I don't really have time to do a KIP to move it back (or just delete it outright) but we should really do that. Maybe we can file a newbie ticket and someone will pick it up..